docs/guides/integration-prebid.md

```diff
@@ -1,172 +1,230 @@
 
-You must provide the Prebid module with the full JSON response body from the applicable endpoint:
+Once configured, the UID2 module will generate a UID2 token for the user and store it in the user's browser. The module will automatically refresh the token as required while your site is open in the user's browser.
 
-- [POST /token/generate](../endpoints/post-token-generate.md) for a new UID2 token.
-- [POST /token/refresh](../endpoints/post-token-refresh.md) for a refreshed UID2 token.
+The user's DII can be passed to the UID2 module hashed or unhashed. If the DII is passed to the UID2 module unhashed, the module will hash it for you. If the DII is passed to the module hashed, it must be normalized before hashing. See [Normalization and Encoding](../getting-started/gs-normalization-encoding.md) for more details.
 
-For an example, see [Sample Token](#sample-token).
+The UID2 module will encrypt the hashed DII before sending it to the UID2 service.
 
-As long as the refresh token remains valid, the module refreshes the UID2 token as needed.
+The following sections demonstrate the different ways to configure the UID2 module and list the requirements for the DII passed to the module.
 
-### Response Storage Options
+:::note
+The examples assume you are using the UID2 production environment. During integration testing use the UID2 integration environment by setting `params.uid2ApiBase` to `"https://operator-integ.uidapi.com"`. You will have a different **subscription ID** and **public key** for the integration environment.
+:::
 
-When you configure the module to use Client Refresh mode, you must choose **one** of the following options for storing the API response information.
+### Configure for email address
 
-| Option | Details | Use Case | 
-| --- | --- | --- |
-| Set `params.uid2Cookie` to the name of the cookie that contains the response body as a JSON string. | See [Client Refresh Cookie Example](#client-refresh-cookie-example). | Do not choose this option unless you are sure there's enough space left in your cookie to store the response body. |
-| Set `params.uid2Token` to the response body as a JavaScript object. | See [Client Refresh uid2Token Example](#client-refresh-uid2token-example). | You might choose to provide the response body via `params.uid2Token` in either of these cases:<br/>- If storing the response body on the cookie will exceed the cookie size limit.<br/>- If you prefer to have the flexibility to manage the storage of the response body yourself. |
+Configure the UID2 module with an email address:
 
-### Client Refresh Cookie Example
+```js
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        serverPublicKey: publicKey,
+        subscriptionId: subscriptionId,
+        email: 'user@example.com',
+      }
+    }]
+  }
+});
+```
 
-In this example, the cookie is called `uid2_pub_cookie`.
+No normalization or hashing is required by the publisher.
```

Comment:
I know you are saying its required, but want to point out that again what is accepted, something similar to "You have the option to pass in a raw or hashed DII. If you pass in raw, the UID2 module will normalize and hash the email address before sending the encrypted hash to the UID2 service."

Reply:
Added a list of the options

Reply:
Super clear now, thanks!